elementary/files
# Fix overlaybar in column view

Based on https://github.com/elementary/files/pull/210 which should be reviewed first.

The overlay status bar does not appear when in column view although this is not blocked by ViewContainer.  It seems reasonable that it should be shown as otherwise only the name is shown in Column View, as in Icon View.

This fixes the issue by having MillerView repeat the hover signal of the active slot.

TO TEST:
* Check the behaviour and appearance of the overlay status bar when in Column View.
* There is no expectation of regressions in other behaviours.
